packages/node_modules/@webex/plugin-meetings/README.md

```diff
@@ -51,6 +51,20 @@ webex.meetings.unregister()
 
 #### Creating a basic meeting
 
+##### Via hydra room id
```

Comment:
Since "hydra" is an internal service name, external developers may not know what this means.

```suggestion
##### Via [developer portal room id](https://developer.webex.com/docs/api/v1/rooms)
```

Reply:
will do